src/main.rs

```diff
@@ -53,12 +53,12 @@ impl cli::Worker for Worker {
 
 fn main() -> Result<(), cli::error::Error> {
 	let version = VersionInfo {
-		name: "Parity Polkadot",
+		name: "Parity Polkadot Client.",
 		commit: env!("VERGEN_SHA_SHORT"),
 		version: env!("CARGO_PKG_VERSION"),
 		executable_name: "polkadot",
-		author: "Parity Team <admin@parity.io>",
-		description: "Polkadot Relay-chain Client Node",
+		author: "Wood/Habermeier/Paronyan/Pepyakin/Köcher/Drwięga et al.",
```

Comment:
Thanks! :+1:

Reply:
I also don't see why we remove parity team as author?